### ISSUE TITLE
Add last update value to json

### ISSUE THREAD

**Issue by Simperfy** [Simperfy/Covid-19-API-Philippines-DOH#2]

**Comment by roirotoras012** [comment]
sir can I use your api for my mobile app? i need data for it :(

**Comment by Simperfy** [comment]
> 
> 
> sir can I use your api for my mobile app? i need data for it :(

Of course sir, in fact me in my colleagues are going to use this API for our upcoming mobile app too.

**Comment by Simperfy** [comment]
We're still adding more endpoints and the data will also include information about the hospitals in the philippines (before end of august).
The Information about hospitals will include the vacant/occupied status of ICU, Isolation Bed, Bed Wards, Mechanical Vent, and location information such as longitude and latitude.

If you found a bug or have a suggestion feel free to post an issue.

**Comment by roirotoras012** [comment]
seems like the other endpoitns doesnt return the exact number of cases sir . the api/summary is not right.
can i use this ? https://covid19-api-philippines.herokuapp.com/api/filter/region_res/Region%20X:%20Northern%20Mindanao

**Comment by roirotoras012** [comment]
it also doesnt return right

**Comment by Simperfy** [comment]
> 
> 
> seems like the other endpoitns doesnt return the exact number of cases sir . the api/summary is not right.

The api/summary should be fixed by now.

> can i use this ? https://covid19-api-philippines.herokuapp.com/api/filter/region_res/Region%20X:%20Northern%20Mindanao
> it also doesnt return right

Cleared the cache, this should be returning the correct numbers now.


I appreciate the concern ❤️, please post new issue if there's still a problem.

**Comment by roirotoras012** [comment]
thank you very much sir.
I will gladly inform you if ever there's an issue.

**Comment by roirotoras012** [comment]
sir can i ask why there are some instances that it will say "been blocked by cors policy" ? sorry im just a newbie haha

**Comment by Simperfy** [comment]
Hi @roirotoras012 cors is enabled in the server
Upon reviewing the requests you made on the server, you mistakenly set "method=OPTIONS" instead of "method=GET". Please set the method to GET when performing request to avoid getting "been blocked by cors policy".

**Comment by roirotoras012** [comment]
okay done sir.
 i think its a bug? cause at first i cant get the data but then after some minute i can already get data

**Comment by Simperfy** [comment]
> 
> 
> okay done sir.
> i think its a bug? cause at first i cant get the data but then after some minute i can already get data

Can confirm it's a bug when you visit "api/get". 
I Appreciate the feedback @roirotoras012, I'm working to fix the issue.

If you found similar bugs please open new issue like this -> #10

**Comment by Simperfy** [comment]
Added last_update to response json at 6050bd3
